src/utils/accountService.js

```diff
@@ -0,0 +1,353 @@
+      await this._generateED25519(true, balance, num, startup)
+    ]).then((allResponses) => {
+      const ecdsaResponses = allResponses[0];
+      const aliasEcdsaResponses = allResponses[1];
+      const ed25519Responses = allResponses[2];
+      
+      this._logAccountTitle();
+      ecdsaResponses.forEach(element => {
+        this._logAccount(element.accountNum, element.balance, element.wallet._signingKey().privateKey);
+      });
+      this._logAccountivider();
+
+      this._logAliasAccountTitle();
+      aliasEcdsaResponses.forEach(element => {
+        this._logAliasAccount(element.accountNum, element.balance, element.wallet);
+      });
+      this._logAliasAccountDivider();
+
+      this._logAccountTitle();
+      ed25519Responses.forEach(element => {
+        this._logAccount(element.accountNum, element.balance, element.wallet._signingKey().privateKey);
+      });
+      this._logAccountivider();
+    })
+  }
+
+  /**
+   * @internal
+   * Generate ECDSA accounts.
+   * @param {boolean} async
+   * @param {number} balance
+   * @param {number} num
+   * @param {boolean} startup
+   */
+  async _generateECDSA(async, balance, num, startup) {
+    let ecdsaAccountNumCounter = 1002;
+    const params = [];
+
+    if (!async) this._logAccountTitle();
+
+    for (let i = 0; i < num; i++) {
+      let wallet = hethers.Wallet.createRandom();
+      if (startup && this.privateKeysECDSA[i]) {
+        wallet = new hethers.Wallet(this.privateKeysECDSA[i]);
+      }
+      params.push(await this._createAccount(
+        async,
+        ecdsaAccountNumCounter++,
+        balance,
+        startup,
+        wallet
+      ));
+    }
+
+    if (!async) {
+      this._logAccountivider();
+    }
+    return params;
+  }
+
+  /**
+   * @internal
+   * Generate Alias ECDSA accounts.
+   * @param {boolean} async
+   * @param {number} balance
+   * @param {number} num
+   * @param {boolean} startup
+   */
+  async _generateAliasECDSA(async, balance, num, startup) {
+    let aliasedAccountNumCounter = 1012;
+    const params = [];
+
+    if (!async) this._logAliasAccountTitle();
+
+    for (let i = 0; i < num; i++) {
+      let wallet = ethers.Wallet.createRandom();
+      if (startup && this.privateKeysAliasECDSA[i]) {
+        wallet = new ethers.Wallet(this.privateKeysAliasECDSA[i]);
+      }
+
+      let accountId = PublicKey.fromString(
+        wallet._signingKey().compressedPublicKey.replace("0x", "")
+      ).toAccountId(0, 0);
+      const transferTransaction = new TransferTransaction()
+        .addHbarTransfer(accountId, new Hbar(balance))
+        .addHbarTransfer(AccountId.fromString("0.0.2"), new Hbar(-balance));
+      const tx = await transferTransaction.execute(this.client);
+      let accountNum = `0.0.${aliasedAccountNumCounter}`;
+      if (startup) {
+        aliasedAccountNumCounter++;
+      } else {
+        await tx.getReceipt(this.client);
+
+        const accountInfo = await new AccountInfoQuery({
```

Comment:
similar to the comment in `_generateECDSA`

If `async`then params should be an array of promises and after the for loop `await Promise.all(params)` and extract `accountNum`, `wallet` and `balance` from the tx.receipt.